common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -38,24 +38,19 @@ import whisk.core.entity.size.SizeOptionString
  * For Swift actions, the source code to execute the action is required.
  * For Java actions, a base64-encoded string representing a jar file is
  * required, as well as the name of the entrypoint class.
- *
- * exec: { kind  : one of supported language runtimes
- *         code  : code to execute if kind is supported
- *         image : container name when kind is "blackbox",
- *         main  : a fully-qualified class name when kind is "java" }
+ * An example exec looks like this:
+ * { kind  : one of supported language runtimes
+ *   code  : code to execute if kind is supported
+ *   image : container name when kind is "blackbox",
+ *   main  : a fully-qualified class name when kind is "java" }
```

Comment:
That's not accurate anymore (it's on me, though). It should say something like: "The name of the entrypoint function, when using a non-default value. For Java, the name of the main class."